.github/workflows/unmarked_tests.yaml

```diff
@@ -0,0 +1,46 @@
+name: Unmarked Tests Checker
+
+on:
+  workflow_call:
+    inputs:
+      project_root:
+        required: true
+        type: string
+
+permissions:
+  contents: read
+
+jobs:
+  pytest-linux:
+      runs-on: ubuntu-latest
+
+      steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.10"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pytest-custom-exit-code
+          pip install .
```

Comment:
Can we call the actions/pytest-setup rather than doing these steps

Reply:
And then just call pip install pytest-blahblah in this file

Reply:
I could have but I decided against it because that would still take two steps to get all the dependencies/python installed but it would also install pytest twice essentially where the regular pytest from the action would not be needed or used.